MyString.java

```diff
@@ -36,7 +56,29 @@ public static int countChar(String str, char ch) {
      * @return true is str1 is a subset of str2, false otherwise
      */
     public static boolean subsetOf(String str1, String str2) {
-         //// Replace the following statement with your code
+        if (str1 == "") {
+            return true;
+        }

```

Comment:
Use either `str.equals("")` or `str.length == 0`. 
using == for String comparison is not consistent